ui/app/components/clients/dashboard.js

```diff
@@ -161,6 +184,25 @@ export default class Dashboard extends Component {
     });
   }
 
+  @action
+  handleEditStartMonth(month, year) {
+    if (!month && !year) {
+      // reset and do nothing. They pressed cancel
+      this.startMonth = this.startYear = null;
```

Comment:
Are you intending to set `startMonth` to a boolean here?

Reply:
At the beginning, via a tracked property startMonth is null, and in this situation, I was trying to prevent them from saving a new startMonth if they hadn't selected one. However, I solved for this by disabling save unless they have a startMonth and startYear. So I can remove this. Thanks for the question!